lib/fastly_nsq/messenger.rb

```diff
@@ -1,20 +1,20 @@
+# frozen_string_literal: true
+
 module FastlyNsq::Messenger
-  DEFAULT_ORIGIN = 'Unknown'.freeze
+  DEFAULT_ORIGIN = 'Unknown'
   @originating_service = DEFAULT_ORIGIN
 
   module_function
 
-  def deliver(message:, on_topic:, originating_service: nil, meta: {})
+  def deliver(message:, topic:, originating_service: nil, meta: {})
```

Comment:
does this need to change?

Reply:
It does not.  I took the opportunity to smooth out the inconsistencies in the named parameters, so this matches `FastlyNsq::Listener`, `FastlyNsq::Consumer` and `FastlyNsq::Producer`.

Reply:
https://github.com/fastly/fastly_nsq/pull/65/commits/fd4e1d784cda1eb92b7353f203fd062e09d7cf1a

Reply:
This is another thing that is going to need to be updated in a lot of places.
eg: Northstar doesn't have a listener ATM but it does use `Messenger` so, with this change we will also likely want to update Northstar right away just for this (so we don't forget about it later :P )

Reply:
Hopefully `find . -iname '*.rb' | xargs sed -i '' 's/on_topic:/topic:'` will work